recipes/libmagic/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{%set name = "libmagic" %}
+{%set version = "5.30" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "9cb61c1399eea2633688e0c2187b67f695210074fcbce2cd1f0701a2792f50dd" %}
+
+package:
+  name: libmagic
+  version: {{ version }}
+
+source:
+  fn: FILE{{ version.replace(".", "_") }}.tar.gz
+  url: https://github.com/file/file/archive/FILE{{ version.replace(".", "_") }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - automake
+    - autoconf
+    - m4
+    - libtool
+    - toolchain
+
+test:
+  commands:
+    - "file --help &> /dev/null"
+    - "ls $CONDA_PREFIX/lib/libmagic.so"  # [linux]
```

Comment:
Not sure about this one - I've seen both used, both are testing for file presence and read-access.